tests/testthat/_snaps/github-actions.md

```diff
@@ -36,5 +36,5 @@
     Condition
       Error in `check_uses_github_actions()`:
       x Cannot detect that package {TESTPKG} already uses GitHub Actions.
-      i Do you need to run `use_github_action()`?
+      i Do you need to run `(use_github_action())[usethis::use_github_action()]`?
```

Comment:
Is this really how this should appear in a snapshot?

Reply:
I made a mistake in the order I did (text)[url] instead of  [text](url).  noob.